Bookshelf/shared/build.gradle.kts

```diff
@@ -1,11 +1,14 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
-    kotlin("multiplatform")
+    alias(libsx.plugins.kotlinMultiplatform)
+    alias(libsx.plugins.androidLibrary)
+    alias(libsx.plugins.kotlinSerialization)
+    // For some reason libsx.plugins.realm does not resolve directly so go through the provider
+    id(libsx.plugins.realm.get().pluginId)
+    // For some reason this does not resolve even though [id: 'org.jetbrains.kotlin.native.cocoapods', version: '2.0.0'] is available in Gradle plugin portal
+    // alias(libsx.plugins.cocoapods)
     kotlin("native.cocoapods")
```

Comment:
Could it be added to the versions catalog?

Reply:
No. The comment above states why.